### ISSUE TITLE
Sometimes comments are tried to be parsed as Jinja2 expressions

### ISSUE THREAD

**Issue by felixfontein** [ansible/ansible-lint#2318]
##### Summary
If I start a comment with `noqa`, it seems to be interpreted with Jinaj2:

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

Latest devel versions.

##### OS / ENVIRONMENT

Linux

##### STEPS TO REPRODUCE

Run ansible-lint on the following playbooks:

```console (paste below)
- hosts: localhost
  tasks:
    - name: Foo
      # noqa - {{item...`
      debug:
        msg: nothing
```

or

```console (paste below)
- hosts: localhost
  tasks:
    - name: Foo
      # noqa - {{item...
      debug:
        msg: nothing
```

##### Desired Behavior

Does not report errors for Jinja2 expressions.

##### Actual Behavior

The first playbook reports (among expected things)
```
jinja: Syntax error in jinja2 template: {{item...` (jinja[invalid])
test-ansible-lint.yml:3 unexpected char '`' at 9
```

The second playbook reports (among expected things):
```
jinja: Jinja2 spacing could be improved: {{item... -> {{ item... (jinja[spacing])
test-ansible-lint.yml:3 Jinja2 template rewrite recommendation: `{{ item...`.
```
